src/containers/Dashboard/Forms/TutorialForm.js

```diff
@@ -72,15 +74,23 @@ class TutorialForm extends Component {
         courseId,
         tags
       };
-      this.setState({
-        name: "",
-        content: "",
-        courseId: "",
-        tags : [],
-      })
+      
       API.tutorialData(data, result => {
         if (result.status === "201") {
-          this.formRef.reset();
+          self.setState({
```

Comment:
create a function for that to reset the values not here

Reply:
Done